### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by alphaolomi**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -102,3 +102,7 @@ dist
 
 # TernJS port file
 .tern-port
+
+yarn.lock
+
+package-lock.json
```
Comment (alphaolomi) [inline .gitignore:108]: Hey everyone

I understand the project uses npm and am some  use use yarn

Due to differences in package manager I ask we keep the lock files ignored

Due to the nature of the project not keeping the lock is not fatal

Comment (Kijacode) [reply .gitignore:108]: Bro @gsonly , what are you think about this, am okay with this

Comment (gsonly) [reply .gitignore:108]: I also prefer yarn... its fine by me.

**Review by tr1ms — APPROVED** [review]
:+1:

**Review by alphaolomi — COMMENTED** [review]
@gsonly @Kijacode 

I think this PR is ready

**Pull request merged**